src/components/Calculator.js

```diff
@@ -46,9 +59,20 @@ class Calculator extends Component {
     return (
       <>
         <Screen value={this.getScreenValue()} />
-        <div style={{ display: "flex" }}>
-          <div>
+        <Operator value="clear" onClick={this.handleOperatorClick} />
+        <div style={{
+          display: "flex",
+          flexWrap: "wrap-reverse",
+         }}>
+         <div style={{
+           display: "flex",
+           flexWrap: "wrap-reverse",
+           alignContent: "flex-end",
+           width: "250px",
```

Comment:
```suggestion
           width: 250,
```
Shorthand way to write this when using inline styles